crepecake/src/main/java/com/google/cloud/tools/crepecake/registry/DockerCredentialRetriever.java

```diff
@@ -82,13 +83,27 @@ public Authorization retrieve()
         // Throws an exception if the credential store does not have credentials for serverUrl.
         if (output.contains("credentials not found in native keychain")) {
```

Comment:
can we determine this by exit code?

Reply:
We actually need to find an alternative to using the Docker credential helper method here, as implementations of the credential helper differ. Filed https://github.com/GoogleCloudPlatform/minikube-build-tools-for-java/issues/118